### PR TITLE
feat: Improve guide connecting machines and creating dashboards

### PR DESCRIPTION
**Pull request by oarne97**

### CODE REVIEW

File: docs/content/en/docs/Getting started/connecting-machines-creating-dashboards/index.md [hunk 1]
```diff
@@ -7,99 +7,100 @@ description: >
...
 
-**The basic approach for data processing on the local hardware is to extract data from various data sources (OPC/UA, MQTT, Rest), extract the important information, and then make it available to the United Manufacturing Hub via a predefined interface (MQTT). For this data processing on the local hardware we use Nodered.**
+TODO
+#455
 
-To extract and pre-process the data from different data sources we use the open source software node-red. node-red is a low-code programming for event-driven applications.
+## 1. Introduction
 
-If you haven't worked with node-red yet, [here](https://nodered.org/docs/user-guide/) is a good documentation from node-red!
+UMH Hub has a wide range of ports and connectors.
```
Comment (kruegeralex) [inline docs/content/en/docs/Getting started/connecting-machines-creating-dashboards/index.md:17]: The United Manufacturing Hub has a wide variety of connectors and therefore offers maximum connectivity to different machines and systems.

File: docs/content/en/docs/Getting started/connecting-machines-creating-dashboards/index.md [hunk 2]
```diff
@@ -7,99 +7,100 @@ description: >
...
+#455
 
-To extract and pre-process the data from different data sources we use the open source software node-red. node-red is a low-code programming for event-driven applications.
+## 1. Introduction
 
-If you haven't worked with node-red yet, [here](https://nodered.org/docs/user-guide/) is a good documentation from node-red!
+UMH Hub has a wide range of ports and connectors. 
 
-{{< imgproc nodered Fit "800x800" >}}{{< /imgproc >}}
+{{< imgproc logos_data_sources Fit "800x800" >}}{{< /imgproc >}}
```
Comment (kruegeralex) [inline docs/content/en/docs/Getting started/connecting-machines-creating-dashboards/index.md:19]: This cross out. And the subline "could contain sensitive information".
It is best to use "www.draw.io" for the graphic.
Insert the Weihenstephaner standard logo and the logos from our website in a "chaotic order".

WS
image

File: docs/content/en/docs/Getting started/connecting-machines-creating-dashboards/index.md [hunk 3]
```diff
@@ -7,99 +7,100 @@ description: >
...
 
-[Here you can download the flow](/examples/nodered/standard_flow.json)
+These are:
```
Comment (kruegeralex) [inline docs/content/en/docs/Getting started/connecting-machines-creating-dashboards/index.md:21]: Here are a few selected examples

File: docs/content/en/docs/Getting started/connecting-machines-creating-dashboards/index.md [hunk 4]
```diff
@@ -7,99 +7,100 @@ description: >
...
+- MQTT ([documentation for this node](https://cookbook.nodered.org/mqtt/))
 
-{{< imgproc nodered_general Fit "800x800" >}}{{< /imgproc >}}
+Via Node-RED, these data sources can be read out easily or, in the case of IO-Link, are even read out automatically and made available via MQTT. 
 
-Basically, 3 pieces of information must be communicated to the system. For more information feel free to check [this article](/docs/concepts/mqtt/). These 3 information must be set to the system via the green configuration node-red, so that the data can be assigned exactly to an asset
+Using these data points and our data model, the data can then be converted into a standardized data model. This is done independently of machine type and manufacturer and can be made usable for various data services, e.g. Grafana.
 
-The customer ID to be assigned to the asset: *customerID*
+## 2. Node-RED
```
Comment (kruegeralex) [inline docs/content/en/docs/Getting started/connecting-machines-creating-dashboards/index.md:34]:
```suggestion
## 2. Extracting and preparing data with the help of Node-RED
```

File: docs/content/en/docs/Getting started/connecting-machines-creating-dashboards/index.md [hunk 5]
```diff
@@ -7,99 +7,100 @@ description: >
...
 
-The location where the asset is located: *location*
+To extract and preprocess the data from different data sources, we use the open source software Node-RED. Node-RED is a low-code programming for event-driven applications. 
 
-The name of the asset: *AssetID*
+Originally, Node-RED comes from the area of smart home and programming implementations, but is also used in manufacturing more frequently. For more information feel free to check [this article](https://docs.umh.app/docs/concepts/node-red-in-industrial-iot/).
 
-Furthermore, you will find under the general settings, the state logic which determines the machine *state* with the help of the *activity* and *detectedAnomaly* topic. For more information feel free to check [this article.](/docs/concepts/mqtt/)
+In the following, the procedure for creating a Node-RED flow is described in detail using **two examples**. The first example is about a cutting machine which will be retrofitted by external sensor technology. The second example deals with the connection of already existing sensors from with our system in order to visualize the acquired data.
```
Comment (kruegeralex) [inline docs/content/en/docs/Getting started/connecting-machines-creating-dashboards/index.md:40]:
```suggestion
In the following, the procedure for creating a Node-RED flow is described in detail using **two examples**.

1. Here we will retrofit a cutting machine with the help of external sensors.
2. In the second example, we will use OPC/UA to read a warping machine data directly from the PLC.
```

File: docs/content/en/docs/Getting started/connecting-machines-creating-dashboards/index.md [hunk 6]
```diff
@@ -7,99 +7,100 @@ description: >
...
-- Rest API  ([documentation for this node](https://cookbook.nodered.org/http/create-an-http-endpoint))
-- Modbus  ([documentation for this node](https://flows.nodered.org/node/node-red-contrib-modbus))
-- MQTT ([documentation for this node](https://cookbook.nodered.org/mqtt/))
+You would like to determine the output and machine condition of a cutting machine.
+
+Used Sensors:
 
-**Interaction with sensorconnect (Plug and Play connection of IO-Link Senosors):**
+- Light Barrier
+- Button Bar
```
Comment (kruegeralex) [inline docs/content/en/docs/Getting started/connecting-machines-creating-dashboards/index.md:49]:
```suggestion
- Button Bar
```
```suggestion
- Light Barrier
- Button Bar
- Capacitive sensor
```

File: docs/content/en/docs/Getting started/connecting-machines-creating-dashboards/index.md [hunk 7]
```diff
@@ -7,99 +7,100 @@ description: >
...
 
-With the help of Sensorconnect, different sensors can be connected quickly and easily via an IFM gateway. The sensor values are automatically extracted from the software stack and made available via [MQTT](http://www.steves-internet-guide.com/mqtt-works/).
+With the help of Sensorconnect, sensors can be connected quickly and easily via an IFM gateway. The sensor values are automatically extracted from the software stack and made available via [MQTT](https://docs.umh.app/docs/concepts/mqtt/). 
 
-To get a quick and easy overview of the available MQTT messages and topics we recommend the [MQTT Explorer](http://mqtt-explorer.com/). If you don't want to install any extra software you can use the MQTT-In node to subscribe to all available topics by subscribing to  `#` and then direct the messages of the MQTT in nodes into a debugging node. You can then display the messages in the nodered debugging window and get information about the topic and available data points.
+*1. Connect Sensors*
```
Comment (kruegeralex) [inline docs/content/en/docs/Getting started/connecting-machines-creating-dashboards/index.md:53]:
```suggestion
#### 1. Connect Sensors
```

File: docs/content/en/docs/Getting started/connecting-machines-creating-dashboards/index.md [hunk 8]
```diff
@@ -7,99 +7,100 @@ description: >
...
 
+TODO: Picture or illustration of how to connect sensors
 
-Topic structure: `ia/raw/<transmitterID>/<gatewaySerialNumber>/<portNumber>/<IOLinkSensorID>`
+*2. Make the connected sensors visible to our system*
```
Comment (kruegeralex) [inline docs/content/en/docs/Getting started/connecting-machines-creating-dashboards/index.md:57]:
```suggestion
#### 2. Make the connected sensors visible to our system
```

File: docs/content/en/docs/Getting started/connecting-machines-creating-dashboards/index.md [hunk 9]
```diff
@@ -7,99 +7,100 @@ description: >
...
 
-#### Example for ia/raw/
+Based on an IP address, which is assigned to each sensor (or gateway?), the sensor can be integrated into our system. Only then is it possible to read out sensor values. The adaptation of the IP range is required.
 
-Topic: `ia/raw/2020-0102/0000005898845/X01/210-156`
+TODO: ADAPTION OF IP-RANGE
 
-This means that the transmitter with the serial number `2020-0102` has one ifm gateway connected to it with the serial number `0000005898845`. This gateway has the sensor `210-156` connected to the first port `X01`.
+*3. Creating a flow*
```
Comment (kruegeralex) [inline docs/content/en/docs/Getting started/connecting-machines-creating-dashboards/index.md:63]:
```suggestion
#### 3. Creating a Node-RED flow*
```

File: docs/content/en/docs/Getting started/connecting-machines-creating-dashboards/index.md [hunk 10]
```diff
@@ -7,99 +7,100 @@ description: >
...
 
-```json
-{
-"timestamp_ms": 1588879689394, 
-"distance": 16
-}
-```
+In Node-RED basically three pieces of information must be communicated to the system.
```
Comment (kruegeralex) [inline docs/content/en/docs/Getting started/connecting-machines-creating-dashboards/index.md:71]: In order to contextualise the resulting data points with the help of the United Manufacturing Hub, 3 identifiers are necessary:

File: docs/content/en/docs/Getting started/connecting-machines-creating-dashboards/index.md [hunk 11]
```diff
@@ -7,99 +7,100 @@ description: >
...
 
+- The customer ID to be assigned to the asset: *customerID*
```
Comment (kruegeralex) [inline docs/content/en/docs/Getting started/connecting-machines-creating-dashboards/index.md:73]:
```suggestion
- The customer ID to be assigned to the asset: *customerID* (Default value for the development setup "factoryinsight")
```

File: docs/content/en/docs/Getting started/connecting-machines-creating-dashboards/index.md [hunk 12]
```diff
@@ -7,99 +7,100 @@ description: >
...
 
-### Extract information and make it available to the **outputs**:
-In order for the data to be processed easily and quickly by the United Manufacturing hub, the input data (OPC/UA, Siemens S7) must be prepared and converted into a standardized data format (MQTT Topic). For a deep explanation of our MQTT data model check [here](/docs/concepts/mqtt/) and [here](/docs/concepts/state).
+- The location where the asset is located: *location*
```
Comment (kruegeralex) [inline docs/content/en/docs/Getting started/connecting-machines-creating-dashboards/index.md:75]:
```suggestion
- The location where the asset is located: *location* (Can be chosen freely)
```

File: docs/content/en/docs/Getting started/connecting-machines-creating-dashboards/index.md [hunk 13]
```diff
@@ -7,99 +7,100 @@ description: >
...
 
-{{< imgproc nodered_outputs Fit "800x800" >}}{{< /imgproc >}}
+- The name of the asset: *AssetID*
```
Comment (kruegeralex) [inline docs/content/en/docs/Getting started/connecting-machines-creating-dashboards/index.md:77]:
```suggestion
- The name of the asset: *AssetID* (Can be chosen freely)
```

**Comment by kruegeralex** [comment]
Hi @oarne97 

thanks for your input so far.

Akuell still further remarks:
- I would make it clearer that step 3.1 and 3.2 are the same for all Sebsonren and then different for everything that reads not so good.
- Add a disclaimer (best under the button bar) that the system must still be added analogous to our data model layers.

**Comment by kruegeralex** [comment]
@Scarjit

**Comment by oarne97** [comment]
@kruegeralex

**Review by Scarjit — APPROVED** [review]

**Review by JeremyTheocharis — CHANGES_REQUESTED** [review]
Let's spellcheck it and try to format it better for the eyes, e.g., prevent unnecessary bold texts, consistent use of headings, etc.

**Review by JeremyTheocharis — APPROVED** [review]

**Pull request merged**